metadata/metadata.go

```diff
@@ -90,7 +90,7 @@ func readFile(file string) ([]byte, error) {
 	return ioutil.ReadFile(file)
 }
 
-const RKEVersionDev = "v0.2.3"
+const RKEVersionDev = "v1.1.1-rc100"
```

Comment:
Shouldn't be required after https://github.com/rancher/rke/pull/2051 is merged in